python/cudf/dataframe/string.py

```diff
@@ -108,9 +108,43 @@ def cat(self, others=None, sep=None, na_rep=None):
             (same type as caller) of str dtype is returned.
         """
         from cudf.dataframe import Series, Index
+        def is_list_like(others):
+            if isinstance(others, (Series, list, tuple, Index)) and not isinstance(others, (str, bytes)):
```

Comment:
`Series` and `Index` shouldn’t be included in this isinstance, and without them you can much more easily check if it’s an instance of `collections.abc.Sequence` instead of list and tuple.

Reply:
Removed `Series` and `Index` from here and moved them to line 131 where there is a necessity for the `Series` & `Index` checks to happen.